omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitAttributeTypeController1_9Test.java

```diff
@@ -158,4 +164,16 @@ public void purge_shouldPurgeAVisitAttributeType() throws Exception {
 		Assert.assertNull(service.getVisitAttributeTypeByUuid(visitAttributeTypeUuid));
 		Assert.assertEquals(originalCount - 1, service.getAllVisitAttributeTypes().size());
 	}
+	
+	@Test
+	public void shouldGetAVisitAttributeTypeByUuid() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" +getUuid());
+		SimpleObject result = deserialize(handle(req));
+		
+		VisitAttributeType visitAttributeType = service.getVisitAttributeTypeByUuid(getUuid());
+		assertNotNull(result);
```

Comment:
Do we really need the above assert?

Reply:
After stepping back and thought about the assert, I realized we do not need it and I did the needful!